eclipse-scout-core/src/form/fields/treebox/TreeBox.ts

```diff
@@ -79,7 +79,7 @@ export class TreeBox<TValue> extends LookupBox<TValue> implements TreeBoxModel<T
     this._valueSyncing = true;
     let opts = {
       checkOnlyEnabled: false,
-      checkChildren: false
+      checkChildren: this.tree.autoCheckChildren
```

Comment:
Remove checkChildren here and check for this.tree.autoCheckChildren below. No need to pass the default value of an option